tests/rptest/tests/pandaproxy_test.py

```diff
@@ -1313,6 +1379,65 @@ def test_password_change(self):
         # teardown process in RedpandaService relies on the superuser
         admin.update_user(super_username, super_password, super_algorithm)
 
+    @cluster(num_nodes=3)
+    def test_restart_http_proxy(self):
```

Comment:
Much of this is duplicated, is it possible to move this to `PandaProxyEndpoints`, and factor out the auth to the derived tests?

Reply:
Sure, I may put some of the output checkers into `PandaProxyEndpoints`; however, doing to much refactor could make debugging tests difficult. IMO the duplicate code is OK.

> and factor out the auth to the derived tests?

The auth is already factored out with `kwargs`. I don't see a benefit with more generalizations.